packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -24,7 +30,7 @@ const MainTitleBar = () => {
 
     return (
         <React.Fragment>
-            <WalletsBanner />
+            {is_wallet_enabled && <WalletsBanner />}
```

Comment:
@sergei-deriv I think we need to check for eligibility as well as is_wallet_enabled, am I right?
    const { is_eligible } = useWalletMigration();
```suggestion
            {is_wallet_enabled && is_eligible && <WalletsBanner />}
```

Reply:
@mahdiyeh-deriv we check for eligibility inside of the WalletsBanner